README.md

```diff
@@ -1,9 +1,9 @@
 ![PODMAN logo](logo/podman-logo-source.svg)
 
-# Library and tool for running OCI-based containers in Pods
+# Podman: A tool for managing OCI containers and pods
 
-Libpod provides a library for applications looking to use the Container Pod concept,
-popularized by Kubernetes.  Libpod also contains the Pod Manager tool `(Podman)`. Podman manages pods, containers, container images, and container volumes.
+Podman (the POD MANager) is a tool for managing containers and images, volumes mounted into those containers, and pods made from groups of containers.
+Podman is based on `libpod`, a library for container lifecycle management that is also contained in this repository. The `libpod` library provides APIs for managing containers, pods, container images, and volumes.
```

Comment:
I think it's important to work in the "popularized by Kubernetes" bit in here somewhere